content/articles/Why-and-when-you-should-use-Kubernetes/index.md

```diff
@@ -0,0 +1,52 @@
+![WHY AND WHEN SHOULD WE USE KUBERNETES?](/Why-and-when-you-should-use-Kubernetes/image-name/container.jpg)
```

Comment:
You can place the image link in the section 'Overview of Kubernetes and container orchestration.'